README.TXT

```diff
@@ -1,3 +1,32 @@
+Ultimate Tapan Kaikki
+=====================
+
+This is the legendary 1990's game Ultimate Tapan Kaikki, ported to
+SDL2. It runs at least on macOS, Linux, FreeBSD and Windows.
+
+Suomipelit GitHub: https://github.com/suomipelit
+Suomipelit Slack: https://tinyurl.com/suomipelit-slack

```

Comment:
Could mention what the purpose of this Slack is, and that English is also welcome.

Reply:
Same here :D

Reply:
Purpose of the org added, I think that's enough for the time being. The invitation is now in English, so I guess that suffices?